### PR TITLE
Implementation of issues #1 #2 #3 and #4 in vue.js

### PR DESCRIPTION
**Pull request by haterzlin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Vue.component('attribute-category', {
     props: ['category'],
+    computed: {
+        tooMany: function() {
+            return this.category.sum > (this.category.allowed + 3)
```
Comment (haterzlin) [inline index.js:5]: number 3 should be a constant

File: index.js [hunk 2]
```diff
@@ -12,7 +17,11 @@ Vue.component('attribute-category', {
     },
     template: `
     <div class="AttributeClass">
-    <h3>{{ category.id }} {{ category.sum }} of {{ category.allowed + 3}}</h3>
+    <h3>{{ category.id }}
+    <span 
+        v-bind:class="{red: tooMany}"> 
+        {{ category.sum }}
+    </span> of {{ category.allowed + 3}}</h3>
```
Comment (haterzlin) [inline index.js:24]: ditto

File: index.css [hunk 3]
```diff
@@ -20,4 +20,8 @@ html, body {
 
 .full {
     background-color: black;
+}
+
+.red {
```
Comment (pmlich) [inline index.css:25]: .textred

File: index.css [hunk 4]
```diff
@@ -20,4 +20,8 @@ html, body {
 
 .full {
```
Comment (pmlich) [inline index.css:21]: circleBlack / circeFull

File: index.js [hunk 5]
```diff
@@ -1,5 +1,14 @@
 Vue.component('attribute-category', {
     props: ['category'],
+    methods: {
+        recomputeSum: function() { 
+            newsum = 0
```
Comment (haterzlin) [inline index.js:10]: feedback from pmlich: without var this is global variable and could be changed from different place

**Review by longnicknam3 — APPROVED** [review]

**Pull request merged**